composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "thadafinser/user-agent-parser",
+    "name": "ready2order/user-agent-parser",
```

Comment:
Probably shouldn't be renaming the package?

Reply:
yeah sorry that slipped through. I removed that change :)